### PR TITLE
Note that kubenet is not cross-node

### PR DESCRIPTION
**Pull request by bboreham**
Spurred by [Slack conversation](http://kubernetes.slackarchive.io/sig-network/-/1474926173.000903/1475076317.001015/1475076159001014/), clarify what kubenet does and how it is meant to be used in a multi-node cluster.

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/1341)

### CODE REVIEW

**Comment by dcbw** [comment]
@bboreham maybe add some more explanation of kubenet too, perhaps:

```
Kubenet is intended to be a very basic, simple network plugin.  It does not, of itself, implement more advanced features like cross-node networking or network policy.  It is typically used together with a cloud provider that sets up routing rules for communication between nodes, or in single-user development and testing environments.
```

or something like that?

**Comment by bboreham** [comment]
@dcbw amended

**Comment by freehan** [comment]
Thanks!
LGTM

File: docs/admin/network-plugins.md [hunk 1]
```diff
@@ -40,7 +40,11 @@ The CNI plugin is selected by passing Kubelet the `--network-plugin=cni` command
 
 ### kubenet
 
-The Linux-only kubenet plugin provides functionality similar to the `--configure-cbr0` kubelet command-line option.  It creates a Linux bridge named `cbr0` and creates a veth pair for each pod with the host end of each pair connected to `cbr0`.  The pod end of the pair is assigned an IP address allocated from a range assigned to the node either through configuration or by the controller-manager.  `cbr0` is assigned an MTU matching the smallest MTU of an enabled normal interface on the host.  The kubenet plugin is currently mutually exclusive with, and will eventually replace, the --configure-cbr0 option.  It is also currently incompatible with the flannel experimental overlay.
+Kubenet is intended to be a very basic, simple network plugin, on Linux only.  It does not, of itself, implement more advanced features like cross-node networking or network policy.  It is typically used together with a cloud provider that sets up routing rules for communication between nodes, or in single-node environments.
```
Comment (steveperry-53) [inline docs/admin/network-plugins.md]: Suggestion: Change "is intended to be" to "is".

File: docs/admin/network-plugins.md [hunk 2]
```diff
@@ -40,7 +40,11 @@ The CNI plugin is selected by passing Kubelet the `--network-plugin=cni` command
...
+
+Kubenet creates a Linux bridge named `cbr0` and creates a veth pair for each pod with the host end of each pair connected to `cbr0`.  The pod end of the pair is assigned an IP address allocated from a range assigned to the node either through configuration or by the controller-manager.  `cbr0` is assigned an MTU matching the smallest MTU of an enabled normal interface on the host.
+
+The kubenet plugin is currently mutually exclusive with, and will eventually replace, the --configure-cbr0 option. It is also currently incompatible with the flannel experimental overlay.
```
Comment (steveperry-53) [inline docs/admin/network-plugins.md]: Suggestion: Avoid "currently". Avoid talking about the future state of the product.
The kubenet plugin is mutually exclusive with the --configure-cbr0 option. It is also incompatible with the flannel experimental overlay.

**Comment by rothgar** [comment]
As the instigator of this PR I agree with @steveperry-53's suggestions. With those changes it's more clear and LGTM.

**Comment by bboreham** [comment]
I pushed a commit with the requested changes, and removing mention of "flannel experimental overlay" which has been removed from the code.

**Pull request merged**